docs/FHIR-anonymization.md

```diff
@@ -238,6 +240,36 @@ Here is a sample configuration for R4:
 |Empty or Null| The configuration file targets the same FHIR version as the executable.
 |Other values| Other values will raise an exception.
 
+### Processing Errors Specification
+
+Anonymization engine will throw three main exceptions in the program: _AnonymizationConfigurationException_, _AnonymizationProcessingException_ and _InvalidInputException_.
+|Exception|Description|
+|-----|-----|
+|AnonymizationConfigurationException|Raised when configuration file has invalid format or value.|
+|AnonymizationProcessingException|Raised during the process of anonymizing a FHIR node.|
+|InvalidInputException|Raised by invalid format of input FHIR resources.|
```

Comment:
Will we throw `AnonymizerRuleNotApplicableException` (or is it just an internal exception)?

Reply:
Thanks, added this exception here.